src/main/java/core/basesyntax/converter/DataConverter.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.converter;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+
+public interface DataConverter {
+    List<FruitTransaction> covertToTransaction(List<String> data);
```

Comment:
There is a typo in the method name `covertToTransaction`, it should be `convertToTransaction`. Correct naming is crucial for code readability and maintainability.